eq-author/src/App/page/Logic/SkipLogic/SkipLogicPage/SkipLogicEditor/SkipConditionEditor/index.js

```diff
@@ -0,0 +1,153 @@
+/* eslint-disable import/no-unresolved */
+import React from "react";
+import styled from "styled-components";
+import PropTypes from "prop-types";
+import { TransitionGroup } from "react-transition-group";
+import { flow, get } from "lodash/fp";
+
+import Transition from "App/page/Logic/Routing/Transition";
+import Button from "components/buttons/Button";
+
+import { colors } from "constants/theme";
+
+import { RADIO } from "constants/answer-types";
+
+import BinaryExpressionEditor from "App/page/Logic/BinaryExpressionEditor";
+
+import withDeleteSkipCondition from "./withDeleteSkipCondition";
+import withDeleteSkipConditions from "./withDeleteSkipConditions";
+
+import { Label } from "components/Forms";
+
+export const LABEL_IF = "IF";
+export const LABEL_AND = "AND";
+export const LABEL_OR = "OR";
+export const LABEL_GROUP_TITLE = "Skip this quetion";
+export const LABEL_REMOVE_GROUP = "Remove OR";
+export const LABEL_REMOVE_ALL_GROUPS = "Remove logic rule";
+
+const Expressions = styled.div`
+  background: white;
+  padding-top: 1em;
+  border-left: 1px solid ${colors.lightMediumGrey};
+  border-right: 1px solid ${colors.lightMediumGrey};
+`;
+
+const SkipCondition = styled.div``;
+
+export const Title = styled.h2`
+  letter-spacing: 0.05em;
+  font-size: 0.9em;
+  font-weight: bold;
+`;
+
+const Header = styled.div`
+  background: ${colors.lightMediumGrey};
+  padding: 0.5em 1em;
+  margin-top: -1px;
+  border-top: 3px solid ${colors.primary};
+  display: flex;
+  align-items: center;
+`;
+
+const Middle = styled.div`
+  background: ${colors.lightMediumGrey};
+  padding: 0.5em 1em;
+  margin-top: -1px;
+  border-left: 3px solid ${colors.primary};
+  display: flex;
+  align-items: center;
+`;
+
+const RemoveSkipConditionButton = styled(Button).attrs({
+  variant: "tertiary",
+  small: true,
+})`
+  margin-left: auto;
+  padding: 0.2em;
+`;
+
+export class UnwrappedSkipConditionEditor extends React.Component {
+  static propTypes = {
+    pageId: PropTypes.string,
+    expressionGroup: PropTypes.object, // eslint-disable-line
```

Comment:
We should probably look into disabling this eslint rule as we use `PropTypes.object` quite a bit. I think we can add a config file specifying which rules we want.

Reply:
Agreed